azurerm/resource_arm_application_gateway.go

```diff
@@ -12,6 +10,7 @@ import (
 	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
 	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
 	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+	"log"
```

Comment:
could we goimports this file? `log` should be above

Reply:
Done - thanks for the tip

Reply:
Done